src/isContrastAccessible.README.md

```diff
@@ -0,0 +1,19 @@
+# `isContrastFriendly()`
+
+Returns: `{ AA: boolean, AAA: boolean }`
+
+Compares foreground and background colours and returns if these meet [`AA` and `AAA` WebAIM guidelines](https://webaim.org/resources/contrastchecker/).
+
+## Example
+
+```typescript
+import { isContrastFriendly } from '@guardian/libs';
+
+const seagreen = 'seagreen';
+const lightseagreen = 'lightseagreen';
+const blackHex = '#000000';
+
+isContrastFriendly(blackHex, seagreen); // {AA: true, AAA: false}
+isContrastFriendly(blackHex, lightseagreen); // {AA: true, AAA: true}
+isContrastFriendly(blackHex, 'foo bar'); // {AA: false, AAA: false}
```

Comment:
Not sure you need variables whose value is identical to their names.

```suggestion
const blackHex = '#000000';

isContrastFriendly(blackHex, 'seagreen'); // {AA: true, AAA: false}
isContrastFriendly(blackHex, 'lightseagreen'); // {AA: true, AAA: true}

// invalid hexadecimal colours or named colours return false
isContrastFriendly(blackHex, 'foo bar'); // {AA: false, AAA: false}
```

You could also use a variable which has more semantic meaning:

```typescript
const mediumConstrat = ["#000", 'seagreen'] // shorthand hex is supported
const highContrast = ["#000000", 'lightseagreen']

isContrastFriendly(...mediumContrast); // {AA: true, AAA: false}
isContrastFriendly(...highContrast); // {AA: true, AAA: true}
```